src/user/proposals/ProposalDiscussion/DiscussionContent/DiscussionContent.js

```diff
@@ -96,104 +174,118 @@ class DiscussionContent extends Component {
-                labore officia cupidatat. Proident amet minim cupidatat proident
-                velit in ea sint velit. Duis adipisicing excepteur cupidatat
-                consequat ex non cupidatat ea non. Commodo ad anim.
-              </p>
+              <Editor
+                value={this.state.markdownString}
+                disabled={true}
+                apiKey="lvp9xf6bvvm3nkaupm67ffzf50ve8femuaztgg7rkgkmsws3"
+                init={{
+                  height: "100%",
+                  width: "100%",
+                  menubar: false,
+                  plugins: [
+                    "advlist autolink lists link image charmap print preview anchor",
+                    "searchreplace visualblocks code fullscreen",
+                    "insertdatetime media table paste code help wordcount",
+                  ],
+                  toolbar: false,
+                }}
+                onEditorChange={this.handleTinyEditorChange}
+              />
             </div>
             <div className="attached-images">
               <div className="images-title">Attached Images</div>
-              <Row>
-                <Image src={eventImg} rounded className="image-item" />
-                <Image src={eventImg} rounded className="image-item" />
-              </Row>
+              <div
+                style={{
+                  overflow: "hidden",
+                  marginLeft: "2",
+                  marginRight: "2",
+                }}
+              >
+                {images.map((item, index) => {
+                  return (
+                    <div
+                      style={{
+                        boxSizing: "border-box",
+                        float: "left",
+                        margin: "2px",
+                        marginRight: "10px",
+                        overflow: "hidden",
+                        paddingBottom: "16%",
+                        position: "relative",
+                        width: `calc(25% - ${2 * 2}px)`,
+
+                        "&:hover": {
+                          opacity: 0.9,
+                        },
+                      }}
+                    >
+                      <img
+                        onClick={this.toggleModal}
+                        src={item.source}
+                        style={{ maxWidth: "100%", position: "absolute" }}
+                      />
+                    </div>
+                  );
+                })}
+              </div>
+
+              <ModalGateway>
+                {imageModalOpen ? (
+                  <Modal onClose={this.toggleModal}>
+                    <Carousel views={this.state.images} />
+                  </Modal>
+                ) : null}
+              </ModalGateway>
             </div>
           </div>
           <div className="comments">
-            <DiscussionComments commentItems={this.state.comments} />
+            <DiscussionComments
```

Comment:
Better to use destructuring

Reply:
Do we get advantages by destructuring here @Rupeshiya ?

Reply:
Not for now, but think if you want to use other state property in the component, then what will you prefer everytime writing this.state.property or only property after destructuring??
@AuraOfDivinity